save-frontend/src/main/kotlin/org/cqfn/save/frontend/App.kt

```diff
@@ -144,8 +160,9 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
                                 element = buildElement {
                                     child(withRouter { _, params ->
                                         child(HistoryView::class) {
+                                            getOrganization(params["owner"])
```

Comment:
This shouldn't be called from `App`, instead API request should be inside `HistoryView`'s `componentDidMount`

Reply:
Fixed